client/my-sites/earn/ads/style.scss

```diff
@@ -1,3 +1,5 @@
+@import 'my-sites/stats/stats-module/style.scss';
```

Comment:
i'd prefer we did this by having earn/ads import the stats module that then requires these styles. importing the styles directly is going to lead to duplication in the loaded styles.

Reply:
Though really the question is, why does earn/ads depend on stats styles?

Reply:
It used to be a tab in stats. And then it got moved to a dedicated section.

Reply:
👍